tests/unit/composables/LetoModelizerApi.test.js

```diff
@@ -101,6 +102,28 @@ describe('User Authentication', () => {
     });
   });
 
+  describe('Test function: getLibrary', () => {
+    it('should return the library', async () => {
```

Comment:
not a test if library is not found ?

Reply:
beh it will throw an 404 but it's from api so no need to test it. And it's already test in another test file.